infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/Constants.java

```diff
@@ -50,5 +50,17 @@
   public static final String POD_STATUS_PHASE_FAILED = "Failed";
   public static final String POD_STATUS_PHASE_SUCCEEDED = "Succeeded";
 
+  /** The projects volume has a standard name used in a couple of locations. */
+  public static final String PROJECTS_VOLUME_NAME = "projects";
```

Comment:
I think https://github.com/eclipse/che/blob/aaa8f424cd198e68e128bca939d9ac058b9ea52e/wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java#L22 would be a good place for this constant

Reply:
:+1: